core/controllers/topic_editor.py

```diff
@@ -541,14 +544,29 @@ def get(self, topic_id: str) -> None:
         skill_creation_is_allowed = (
             role_services.ACTION_CREATE_NEW_SKILL in self.user.actions)
 
+        curriculum_admin_usernames = (
+            user_services.get_usernames_by_role('ADMIN'))
+
         self.values.update({
-            'classroom_url_fragment': classroom_url_fragment,
+            'classroom_url_fragment': (
+                None if (
+                    str(constants.CLASSROOM_URL_FRAGMENT_FOR_UNATTACHED_TOPICS)
+                    == classroom_url_fragment
```

Comment:
Almost always it's \<variable\> == \<const\>, not \<const\> == \<variable\>.

Reply:
Done.